src/identity/apis/auth.ts

```diff
@@ -316,5 +316,9 @@ export const updateUserAccount = async (accountId, name) => {
     throw new ServerError(response.data.message)
   }
 
-  return response.data
+  // TODO: Remove the following line of code when this PR(https://github.com/influxdata/quartz/issues/6846) is done
+  //       Instead return response.data.
+  const data = {...response.data, id: parseInt(`${response.data.id}`, 10)}
```

Comment:
Would it be easier to just set `id` to `Number(response.data.id)` --  instead of using `parseInt`, stringifying the value, and passing passing base 10 as a param?

Reply:
ah I hadn't used this function before. this is a neato. Thanks!